src/span.js

```diff
@@ -137,13 +137,43 @@ export default class Span {
     }
 
     /**
+     * Checks whether or not a span can be written to.
+     * A span is finalized when any of these happens:
+     * - it has inherited the sampling decision from its parent
+     * - its debug flag is set via the sampling.priority tag
+     * - it is finish()-ed
+     * - setOperationName is called
+     * - it is used as a parent for another span
+     * - its context is serialized using injectors
```

Comment:
the definition of what "finalized" means does not belong to this method's comment. It should be moved to the field definition in span context (where it is currently incomplete).